LICENSE

```diff
@@ -419,8 +419,8 @@ The externally maintained libraries used by Node.js are:
      #  and others. All Rights Reserved.
      #
      # Project: https://github.com/veer66/lao-dictionary
-     # Dictionary: https://github.com/veer66/lao-dictionary/blob/master/Lao-Dictionary.txt
-     # License: https://github.com/veer66/lao-dictionary/blob/master/Lao-Dictionary-LICENSE.txt
+     # Dictionary: https://github.com/veer66/lao-dictionary/blob/HEAD/Lao-Dictionary.txt
+     # License: https://github.com/veer66/lao-dictionary/blob/HEAD/Lao-Dictionary-LICENSE.txt
```

Comment:
These two come from upstream ICU’s LICENSE (https://github.com/nodejs/node/blob/80098e605c1d535e495438210c661c3e69a24535/deps/icu-small/LICENSE#L288-L289) so will get overwritten the next time `tools/license-builder.sh` is run to regenerate the `LICENSE`.

Reply:
Thanks good catch. I'm not sure how I included those as I was not planning any changes in files within dependencies.